src/app/services/region-select.service.ts

```diff
@@ -0,0 +1,28 @@
+import { Injectable } from '@angular/core';
+import { Subject } from 'rxjs';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class RegionSelectService {
+  private region = 'global';
```

Comment:
this feels a bit like its coming from nowhere, but if the array of regions/countries lived here it would be a little more explicit.

I also think we should map the regions used by the news api to something we use for the app, that way if the api we use changes/we use a different api we only have to change the mapping instead of looking all over the place for bits like this